modules/swagger-codegen/src/main/resources/csharp/ApiClient.mustache

```diff
@@ -49,30 +50,32 @@ namespace {{packageName}}.Client
 
         /// <summary>
         /// Initializes a new instance of the <see cref="ApiClient" /> class
-        /// with default configuration and base path ({{{basePath}}}).
+        /// with default configuration and base path (<see cref="SERVER_DEFAULT_ADDRESS"/>).
         /// </summary>
-        public ApiClient()
-        {
+        public ApiClient() {
+            log.Trace("start");
```

Comment:
@iamkarlson I'm not sure how that link was meant to support your comment. In NLog, this is the relevant source for pulling the method name from stack: https://github.com/NLog/NLog/blob/31fdc39f3ecee901ed359f7cc157244cf0a1f538/src/NLog/LoggerImpl.cs#L77

What you're saying is that implementors should implement this same logic to get the method name from the stack.

The problem is that if I were to implement your logging class as a pass-through to NLog, for example:

```
public partial class Logger 
{
    private static NLog.Logger logger = NLog.LogManager.GetCurrentClassLogger();
   
    partial void TracePartial(string message)
    {
        logger.Trace(message);
    }
}
```

The method name seen by NLog would be TracePartial, not whatever method is calling it.